spec/components/parameters/files/limit.ts

```diff
@@ -0,0 +1,15 @@
+import { ParameterObject } from 'openapi3-ts';
+import { createComponentRef } from '../../../../utils/ref';
+
+const limit: ParameterObject = {
+  name: 'limit',
+  in: 'query',
+  required: true,
+  description: 'The amount of files should be returned',
```

Comment:
melhor incluir máximo na descrição:
`The maximum amount of files to be returned`

E pensando melhor agora, em vez de tornar obrigatório, talvez seja melhor ter um valor padrão:
trocar de `required: true` pra tipo `default: 1000`.

Aí no api-control a gente poria algo tipo `limit: limit ?? 1000`.

E acho que vale colocar um valor mínimo, e talvez até máximo também:
`minimum: 1`
`maximum: 10000`